ckanext/dashboard/blueprints/dashboard.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from ckan import model
+import ckan.plugins as p
+from flask import Blueprint, request, redirect, url_for, flash
+from ckan.logic import NotFound
+from ckan.plugins.toolkit import render
+
+# Importa o define el decorador para restringir el acceso a sysadmins.
+# Puedes definirlo en tu extensión o importarlo si ya lo tienes.
+from ckanext.dashboard.decorators import require_sysadmin_user
+
+log = logging.getLogger(__name__)
+
+dashboard_bp = Blueprint('dashboard_bp', __name__, url_prefix='/dashboard-external')
+
+
+@dashboard_bp.route('/', methods=['GET'], endpoint='dashboard_list')
+@require_sysadmin_user
+def index():
+    """Listar las configuraciones de los dashboards"""
+    log.debug("Listando configuraciones de dashboard")
+    context = {'model': model, 'user': p.toolkit.c.user}
+    try:
+        dashboards = p.toolkit.get_action('dataset_dashboard_list')(context, {})
+    except Exception as e:
+        log.error("Error al listar dashboards: %s", e)
+        dashboards = []
+    return render('dashboard/index.html', extra_vars={'dashboards': dashboards})
+
+
+@dashboard_bp.route('/new', methods=['GET', 'POST'], endpoint='dashboard_new')
+@require_sysadmin_user
+def dashboard_new():
+    """Crear un nuevo dashboard (vista y lógica para la creación)"""
+    # Aquí implementa la lógica para crear un dashboard
+    return render('dashboard/new.html')
+
+
+@dashboard_bp.route('/edit/<package_id>', methods=['GET', 'POST'], endpoint='dashboard_edit')
```

Comment:
```suggestion
@dashboard_bp.route('/edit/<dashboard_id>', methods=['GET', 'POST'], endpoint='dashboard_edit')
```
Use the dashboard ID as reference. This is the real unique field in a dashboard object